### DISCUSSION TITLE
Use yy to copy, but there is no highlighting effect.

### DISCUSSION THREAD

**Discussion by liyincode** [JetBrains/ideavim#781] [Q&A] [answered]
hello, guys.

Use yy to copy, but there is no highlighting effect.

It is a problem recently discovered, previously everything was normal.

**Comment by liyincode** [comment]
![example](https://github.com/JetBrains/ideavim/assets/25540709/a2b2efbe-cf2b-433d-890b-0c3ca4159fd2)

**Reply by citizenmatt** [reply, marked as answer]
Try to enable the `highlightedyank` plugin

https://github.com/JetBrains/ideavim/wiki/IdeaVim-Plugins#highlightedyank

**Reply by liyincode** [reply]
Wow, my problem is solved. Thank you very much!👍
